test/integration/page_objects/page_object.rb

```diff
@@ -0,0 +1,55 @@
+require_relative '../poltergeist_helper'
+
+class PageObject
+
+  TIMEOUT = 50.freeze
+
+  attr_reader :element
+
+  def initialize(element = nil)
+    @element = element
+  end
+
+  def page
+    Capybara.current_session
+  end
+
+  # Don't wait when using Capybara's #find unless specified otherwise.
+  def find(css, params={})
+    default_params = { wait: false }.merge(params)
+    page.find(css, default_params)
+  rescue Capybara::ElementNotFound
+    raise "'#{css}' was not found."
+  end
+
+  #########
+  # Waits #
+  #########
+
+  def wait_for_ajax(time_out = TIMEOUT)
+    time_out.times do
+      page.evaluate_script('jQuery.active').zero? ? true : sleep(0.1)
+    end
+    raise "Ajax never finished running."
+  end
```

Comment:
Nice! I had to redescover wait for ajax at work recently. So nice to have it out the gate!